### PR TITLE
CWS CUJ Tweaks for "I want to understand the developer program policies"

### LINKED ISSUE #1982: [CWS CUJ] I want to understand the developer program policies
[Friction Log](https://docs.google.com/document/d/1Rw6dF8IYGKTxGRME3tg_fjjpNGWGN6GNQlag0fFE7u8/edit#heading=h.j4f4zcsvf3wk)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1982 

Changes proposed in this pull request:

- Convert definition list into Subheaders
- Remove requirements for inline installation
- Add One Stop Support contact form link.
- Add link to the designated field for the Privacy Policy

Staged preview
[Program policies](https://deploy-preview-1983--developer-chrome-com.netlify.app/docs/webstore/program_policies/)
[Developer agreement](https://deploy-preview-1983--developer-chrome-com.netlify.app/docs/webstore/terms/)

### CODE REVIEW

**Review by nilanjansiromani — APPROVED** [review]

File: site/en/docs/webstore/program_policies/index.md [hunk 1]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
+also be easily removable by either adjusting the settings or uninstalling the product altogether.
+Ads may not simulate or impersonate system notifications or warnings.
 
-: Ads must be presented in context or clearly state which product they are bundled with. Ads must
-  also be easily removable by either adjusting the settings or uninstalling the product altogether.
-  Ads may not simulate or impersonate system notifications or warnings.
+### Ad Walls {: #ad-walls }
 
-Ad Walls:
+Forcing the user to click on ads or submit personal information for advertising purposes in orderto fully use an app or extension provides a poor user experience and is prohibited.
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:440]:
```suggestion
Forcing users to click ads or submit personal information for advertising purposes in order to fully use an app or extension provides a poor user experience and is prohibited.
```

File: site/en/docs/webstore/program_policies/index.md [hunk 2]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
-: Forcing the user to click on ads or submit personal information for advertising purposes in order
-  to fully use an app or extension provides a poor user experience and is prohibited.
+### Interfering with Third-party Ads and Websites {: #interfering-ads }
 
-Interfering with Third-party Ads and Websites:
+Ads associated with your product may not interfere with any ads on a third-party website or
+application. You may show ads alongside a third-party website only if all of the following
+criteria are met:
+- This behavior is clearly disclosed to the user.
+- There is clear attribution of the ads' source wherever those ads appear.
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:448]:
```suggestion
- There is clear attribution of the ads' sources wherever those ads appear.
```

File: site/en/docs/webstore/program_policies/index.md [hunk 3]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
-  application. You may show ads alongside a third-party website only if all of the following
-  criteria are met:
+{% Aside %}
 
-  - This behavior is clearly disclosed to the user.
-  - There is clear attribution of the ads' source wherever those ads appear.
-  - The ads do not interfere with any native ads or functionality of the website.
-  - The ads do not mimic or impersonate the native ads or content on the third-party website, and
-    the ads adhere to the content policy on [impersonation and deceptive behavior][19].
+Currently, AdSense may not be used to serve ads in Products, per [AdSense policies][adsense].
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:455]:
```suggestion
Currently, AdSense may not be used to serve ads in products, per [AdSense policies][adsense].
```

File: site/en/docs/webstore/program_policies/index.md [hunk 4]
```diff
@@ -511,53 +519,57 @@ requirements:
...
-Notifications and Appeals
-: In the event that your Product is removed from Chrome Web Store, you will receive an email
-  notification to that effect, with further instructions if applicable. Please verify that the
-  associated publisher account with your Product can receive emails from external parties and not get
-  flagged as Spam to ensure that you receive all communications in a timely manner.
-
+### Repeat abuse {: #repeat-abuse }
+
+Serious or repeated violations of the Chrome Web Store Distribution Agreement or these Program
+Policies will result in the suspension of your developer account, and possibly related developer
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:531]:
```suggestion
policies will result in the suspension of your developer account, and possibly related developer
```

File: site/en/docs/webstore/program_policies/index.md [hunk 5]
```diff
@@ -511,53 +519,57 @@ requirements:
...
+tool][report-content].
+
+### Enforcement Circumvention {: #enforcement-circumvention }
+
+Any attempt to circumvent intended limitations or enforcement actions will result in the immediate
+termination of your developer account, and possibly related developer accounts.
+
+### Notifications and Appeals {: #appeals }
+
+In the event that your Product is removed from Chrome Web Store, you will receive an email
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:545]:
```suggestion
In the event that your product is removed from Chrome Web Store, you will receive an email
```

File: site/en/docs/webstore/program_policies/index.md [hunk 6]
```diff
@@ -511,53 +519,57 @@ requirements:
...
+notification to that effect, with further instructions if applicable. Please verify that the
+associated publisher account with your Product can receive emails from external parties and not get
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:547]:
```suggestion
associated publisher account with your product can receive emails from external parties and not get
```

File: site/en/docs/webstore/program_policies/index.md [hunk 7]
```diff
@@ -511,53 +519,57 @@ requirements:
...
+flagged as Spam to ensure that you receive all communications in a timely manner.
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:548]:
```suggestion
flagged as spam to ensure that you receive all communications in a timely manner.
```

File: site/en/docs/webstore/program_policies/index.md [hunk 8]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
 landing page of your listed developer website. Products that include content that may not be
 suitable for all ages should be marked "Mature" on the [Developer Dashboard][mature-content].
 
-Sexually Explicit Material:
+### Sexually Explicit Material {: #sexually-explicit }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:37]:
```suggestion
### Sexually explicit material {: #sexually-explicit }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 9]
```diff
@@ -23,8 +23,8 @@ Please be aware that these policies pertain to your software, content, and digit
 for use in connection with Google Chrome and distributed via the Chrome Web Store, including
 applications, extensions, and themes, as well as any ads bundled or made available through the
 product. These policies have the same meaning as in the [Google Chrome Web Store Developer
-Agreement][7]. There are additional quality guidelines that apply to [Chrome Apps][8] and [Chrome
-Extensions][9]. Be sure to check back from time to time, as these policies may change.
+Agreement][developer-agreement]. There are additional quality guidelines that apply to [Chrome Apps][section-chrome-apps] and [Chrome
+Extensions][section-extensions]. Be sure to check back from time to time, as these policies may change.
 
 ## Content Policies {: #content_policies }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:29]:
```suggestion
## Content policies {: #content_policies }
```

Comment (jpmedley) [reply site/en/docs/webstore/program_policies/index.md:29]: Headings need to be sentence case. I'll correct what I can. Will you please check these docs headings that aren't visible in the PR?

File: site/en/docs/webstore/program_policies/index.md [hunk 10]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+We don't allow content that contains nudity, graphic sex acts, sexually explicit material, or
+content that drives traffic to commercial pornography sites. We also don't allow content that
+promotes incest, bestiality, necrophilia, or non-consensual sexual acts. Google has a
+zero-tolerance policy against child pornography. If we become aware of content with child
+pornography, we will report it to the appropriate authorities and delete the Google Accounts of
+those involved with the distribution. Content which contains non-sexual nudity - such as artistic,
+educational, scientific, or cultural nudity - is generally allowed, but may impact the visibility
+of your Product.
+
+### Violent or Bullying Behavior {: #bullying }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:48]:
```suggestion
### Violent or bullying behavior {: #bullying }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 11]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+  harassment, even if each individual piece of content is not severe.
 
-  - Content which makes a specific threat of serious harm against an individual person or a defined
-    group of people.
-  - Content whose predominant purpose is to single out another person for abuse, malicious attack,
-    or ridicule. Content that results in the unwanted sexualization of a person, including malicious
-    claims about a person's sexual activities, sexual orientation, or gender identity.
-  - A series of posts/comments/photos that, taken together, clearly have the primary intention of
-    harassment, even if each individual piece of content is not severe.
+### Hate Speech {: #hate-speech }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:61]:
```suggestion
### Hate speech {: #hate-speech }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 12]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
-: We don't allow content advocating against or inciting hatred towards groups of people based on
-  their race or ethnic origin, religion, disability, gender, age, veteran status, nationality,
-  sexual orientation, gender, gender identity, or any other characteristic that is associated with
-  systematic discrimination or marginalization.
+Additionally, the visibility of your Product may be impacted if it contains generally hateful
+content not covered by the above definition.
 
-  Additionally, the visibility of your Product may be impacted if it contains generally hateful
-  content not covered by the above definition.
+### Violent Extremism {: #violent }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:71]:
```suggestion
### Violent extremism {: #violent }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 13]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+We remove content that recruits, fundraises, or promotes violence on behalf of extremist groups
+defined by the US State Department and other international organizations. Violent extremism is
+defined as the use of violence and intimidation in the pursuit of political aims or goals outside
+of societal norms.
 
-: We remove content that recruits, fundraises, or promotes violence on behalf of extremist groups
-  defined by the US State Department and other international organizations. Violent extremism is
-  defined as the use of violence and intimidation in the pursuit of political aims or goals outside
-  of societal norms.
+### Impersonation or Deceptive Behavior {: #impersonation }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:78]:
```suggestion
### Impersonation or deceptive behavior {: #impersonation }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 14]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+  One")
 
-: If your product has a blank description field or is missing an icon or screenshots, it will be
-  rejected. If any of your product's content, title, icon, description, or screenshots contains false
-  or misleading information, we may remove it.
+If your product has a blank description field or is missing an icon or screenshots, it will be
+rejected. If any of your product's content, title, icon, description, or screenshots contains false
+or misleading information, we may remove it.
+
+### Featured Products {: #featured-products }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:111]:
```suggestion
### Featured products {: #featured-products }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 15]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+- Gambling content
+- Extensions whose developers have questionable reputations, such as historically misleading or
+  malicious extensions.
 
-: The Chrome Web Store features products that align with our standards, values, and that we believe
-  will produce valuable user experiences. Certain products that don't meet these standards, but which
-  do not explicitly violate CWS policies - such as VPN extensions and Video Downloaders - may be
-  restricted from feature in the Store, but will still be available to users. For example, the
-  following products are currently not featured in the CWS Store:
+### Intellectual Property {: #intellectual-property }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:132]:
```suggestion
### Intellectual property {: #intellectual-property }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 16]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
-  secret, copyright, and other proprietary rights. We will respond to clear notices of alleged
-  copyright infringement. For more information or to file a DMCA request, use [this tool][10].
-  Additionally, the visibility of your Product may be impacted if we believe it potentially infringes
-  on intellectual property rights.
+We don't allow content or services that facilitate online gambling, including but not limited to
+online casinos, sports betting, lotteries, or games of skill that offer prizes of cash or other
+value.
 
-Illegal Activities:
+### Regulated Goods and Services {: #regulated-goods }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:153]:
```suggestion
### Regulated goods and services {: #regulated-goods }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 17]
```diff
@@ -34,197 +34,194 @@ account that is publicly displayed in Chrome Web Store, including your developer
...
+  extension package
+- Fetching remote resources that are not used to evaluate logic, such as images
+- Performing server-side operations with data (such as for the purposes of
+  encryption with a private key)
+
+If our reviewers are unable to determine the full functionality of your
+extension during the review process, we may reject your submission or remove it
+from the store.
+
+### Prohibited Products {: #prohibited-products }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:216]:
```suggestion
### Prohibited products {: #prohibited-products }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 18]
```diff
@@ -242,31 +239,35 @@ installation tactics will be removed from the Chrome Web Store.
...
 
-For more information about this policy, please see the [Developer FAQ][11].
+For more information about this policy, please see the [Developer FAQ][deceptive-installation].
 
 ## Spam & Placement in the Store {: #spam }
 
 Developers are important partners in maintaining a great user experience in the Chrome Web Store.
 
-**Repetitive Content:** We don't allow any developer, related developer accounts, or their
+### Repetitive Content {: #repetitive-content }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:261]:
```suggestion
### Repetitive content {: #repetitive-content }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 19]
```diff
@@ -276,33 +277,41 @@ context.
...
 
-**Notification Abuse:** We do not allow extensions that abuse, or are associated with abuse, of
+### Notification Abuse {: #notifications }
+
+We do not allow extensions that abuse, or are associated with abuse, of
 notifications by sending spam, ads, promotions, phishing attempts, or unwanted messages that harm
 the user's browsing experience.
 
-**Message Spam:** We don't allow extensions that send messages on behalf of the user without giving
+### Message Spam {: #messages }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:304]:
```suggestion
### Message spam {: #messages }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 20]
```diff
@@ -276,33 +277,41 @@ context.
...
-Guidelines][12].
+Guidelines][websmaster-guide].
 
-For additional information about the spam policy, see the [Spam FAQ][13].
+For additional information about the spam policy, see the [Spam FAQ][faq-spam].
 
-For information about Product ranking, please see [these FAQs][14].
+For information about Product ranking, please see [these FAQs][faq-product-ranking].
 
 ## User Data Privacy {: #userdata }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:316]:
```suggestion
## User data privacy {: #userdata }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 21]
```diff
@@ -312,41 +321,40 @@ the collection, use, and sharing of the data. You must limit your use of the dat
 you disclosed. This policy establishes the Chrome Web Store's minimum user data privacy
 requirements; you or your Product must comply with applicable laws.
 
-Please see this [FAQ][15].
+Please see this [FAQ][faq-user-data].
 
 ### Personal or Sensitive User Data {: #userdata_psud }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:326]:
```suggestion
### Personal or sensitive user data {: #userdata_psud }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 22]
```diff
@@ -312,41 +321,40 @@ the collection, use, and sharing of the data. You must limit your use of the dat
...
+website content and resources, form data, web browsing activity, user-provided content and personal
+communications), then your Product must:
 
-  - Post a privacy policy, and
-  - Handle the user data securely, including transmitting it via modern cryptography.
+- Post a privacy policy, and
+- Handle the user data securely, including transmitting it via modern cryptography.
 
-Privacy Policy Requirements
+#### Privacy Policy Requirements {: #privacy-policy-req }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:338]:
```suggestion
#### privacy policy requirements {: #privacy-policy-req }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 23]
```diff
@@ -312,41 +321,40 @@ the collection, use, and sharing of the data. You must limit your use of the dat
...
+The privacy policy must, together with any in-Product disclosures, comprehensively disclose how your
+Product collects, uses and shares user data, including the types of parties with whom it's shared.
+You must make the the policy accessible by providing a link:
 
-  - In the designated field in the Chrome Web Store Developer Dashboard, and
-  - In the Product's inline installation page (if applicable).
+- In the [designated field][account-settings] in the Chrome Web Store Developer Dashboard.
 
-Prominent Disclosure Requirement
+#### Prominent Disclosure Requirement {: #prominent-disclosure }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:346]:
```suggestion
#### Prominent disclosure requirement {: #prominent-disclosure }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 24]
```diff
@@ -312,41 +321,40 @@ the collection, use, and sharing of the data. You must limit your use of the dat
...
+described prominently in the Product's Chrome Web Store page and user interface, then prior to the
+collection, it must:
 
-  - Prominently disclose how the user data will be used, and
-  - Obtain the user's affirmative consent for such use.
+- Prominently disclose how the user data will be used, and
+- Obtain the user's affirmative consent for such use.
 
-Other Requirements
+#### Other Requirements {: #other-userdata }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:355]:
```suggestion
#### Other requirements {: #other-userdata }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 25]
```diff
@@ -371,9 +379,6 @@ Other Requirements
     </tbody>
   </table>
 
-  If your Product uses inline installation pages, then any reference to the Product's Chrome Web
-  Store page also includes its inline installation pages.
-
 ### Limited Uses of User Data {: #limited_use }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:382]:
```suggestion
### Limited uses of user data {: #limited_use }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 26]
```diff
@@ -407,7 +412,7 @@ apply to scraped content or otherwise automatically gathered user data.
 An affirmative statement that your use of the data complies with the Limited Use restrictions must
 be disclosed on a website belonging to your extension; e.g., A link on a homepage to a dedicated
 page or privacy policy noting: "The use of information received from Google APIs will adhere to the
-[Chrome Web Store User Data Policy][16], including the [Limited Use][17] requirements."
+[Chrome Web Store User Data Policy][section-user-data], including the [Limited Use][section-limited-use] requirements."
 
 ### Use of Permissions {: #permissions }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:417]:
```suggestion
### Use of permissions {: #permissions }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 27]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
 
 Ads are considered part of your Product for purposes of content review and compliance with developer
 terms, and therefore must comply with the above content policies. Ads which are inconsistent with
-the [content rating][18] of your products or extension are also in violation of our developer terms.
+the [content rating][content-rating] of your products or extension are also in violation of our developer terms.
+
+### Ads Context and Attribution {: #ads-context }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:432]:
```suggestion
### Ads context and attribution {: #ads-context }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 28]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
-  also be easily removable by either adjusting the settings or uninstalling the product altogether.
-  Ads may not simulate or impersonate system notifications or warnings.
+### Ad Walls {: #ad-walls }
 
-Ad Walls:
+Forcing the user to click on ads or submit personal information for advertising purposes in orderto fully use an app or extension provides a poor user experience and is prohibited.
 
-: Forcing the user to click on ads or submit personal information for advertising purposes in order
-  to fully use an app or extension provides a poor user experience and is prohibited.
+### Interfering with Third-party Ads and Websites {: #interfering-ads }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:442]:
```suggestion
### Interfering with third-party ads and websites {: #interfering-ads }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 29]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
-  - The ads do not interfere with any native ads or functionality of the website.
-  - The ads do not mimic or impersonate the native ads or content on the third-party website, and
-    the ads adhere to the content policy on [impersonation and deceptive behavior][19].
+Currently, AdSense may not be used to serve ads in Products, per [AdSense policies][adsense].
+
+{% endAside %}
 
-<div class="aside aside--note">Currently, AdSense may not be used to serve ads in Products, per <a href="https://support.google.com/adsense/bin/answer.py?hl=en&amp;answer=48182">AdSense policies</a>.</div>
 
 ## Extensions Quality Guidelines {: #extensions }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:460]:
```suggestion
## Extensions quality guidelines {: #extensions }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 30]
```diff
@@ -422,36 +427,39 @@ or features that have not yet been implemented.
...
 
-Single Purpose:
+### Single Purpose {: #single-purpose }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:462]:
```suggestion
### Single purpose {: #single-purpose }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 31]
```diff
@@ -511,53 +519,57 @@ requirements:
...
+
+Serious or repeated violations of the Chrome Web Store Distribution Agreement or these Program
+Policies will result in the suspension of your developer account, and possibly related developer
+accounts. Additionally, you may be banned from using the Chrome Web Store. In extreme cases, this
+may also result in the suspension of related Google services associated with your Google account.
+Repeated infringement of intellectual property rights, including copyright, will also result in
+account termination. For more information on Google's copyright policies, please use [this
+tool][report-content].
+
+### Enforcement Circumvention {: #enforcement-circumvention }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:538]:
```suggestion
### Enforcement circumvention {: #enforcement-circumvention }
```

File: site/en/docs/webstore/program_policies/index.md [hunk 32]
```diff
@@ -511,53 +519,57 @@ requirements:
...
+
+Any attempt to circumvent intended limitations or enforcement actions will result in the immediate
+termination of your developer account, and possibly related developer accounts.
+
+### Notifications and Appeals {: #appeals }
```
Comment (jpmedley) [inline site/en/docs/webstore/program_policies/index.md:543]:
```suggestion
### Notifications and appeals {: #appeals }
```

File: site/en/docs/webstore/terms/index.md [hunk 33]
```diff
@@ -37,9 +36,9 @@ acquire and maintain a valid Google account.
 ## 2\. Accepting this Agreement {: #accepting }
```
Comment (jpmedley) [inline site/en/docs/webstore/terms/index.md:36]:
```suggestion
## 2\. Accepting this agreement {: #accepting }
```

File: site/en/docs/webstore/terms/index.md [hunk 34]
```diff
@@ -227,7 +226,7 @@ marketing the Google Chrome Web Store, Google products and services, or publiciz
 Google grants to Developer a limited, non-exclusive, worldwide, royalty-free license to use the
 Google Chrome Brand Features for the Term of this Agreement solely for marketing purposes and only
 in accordance with the Google Chrome Brand Guidelines located at
-[https://chrome.google.com/extensions/intl/en/branding.html][12].
+[https://developer.chrome.com/docs/webstore/branding][branding].
 
 ## 7\. Product Takedowns, Review, and Updates {: #review }
```
Comment (jpmedley) [inline site/en/docs/webstore/terms/index.md:231]:
```suggestion
## 7\. Product takedowns, review, and updates {: #review }
```

File: site/en/docs/webstore/terms/index.md [hunk 35]
```diff
@@ -305,7 +304,7 @@ any outstanding policy violations.
 8.3 Google may suspend or terminate your right to publish Products on the Web Store for any reason,
 including but not limited to: (a) violation of the Agreement or the Google Chrome Web Store Program
 Policies, or (b) infringement upon any intellectual property rights, including copyright. For more
-information on Google's copyright policies, use [this tool][15].
+information on Google's copyright policies, use [this tool][google-copyright].
 
 ## 9\. Privacy and Information {: #privacy }
```
Comment (jpmedley) [inline site/en/docs/webstore/terms/index.md:309]:
```suggestion
## 9\. Privacy and information {: #privacy }
```

File: site/en/docs/webstore/program_policies/index.md (hunk 9 above)
Comment (AmySteam) [reply site/en/docs/webstore/program_policies/index.md]: WIll do! 👍

**Review by RobbyLena — COMMENTED** [review]
Hey sry

**Review by jpmedley — APPROVED** [review]

**Pull request merged**